styles.css

```diff
@@ -136,3 +140,107 @@ body {
 .main__button:hover {
   opacity: .5;
 }
+
+@media (max-width:667px) {
+  #checkbox {
+    display: none;
+  }
+
+/* .container, .logo__circle {
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/-Kostikova-Irina/pullRequest?prid=6356170)